src/main/java/de/retest/ui/PathElement.java

```diff
@@ -19,50 +19,42 @@
 	private final String elementName;
 
 	@XmlElement
-	private final String suffix;
+	private final int suffix;
 
-	public PathElement( final String elementName, final String suffix ) {
+	public PathElement( final String elementName, final int suffix ) {
 		if ( elementName == null ) {
 			throw new NullPointerException( "Element name must not be null." );
 		}
 		this.elementName = elementName.trim();
 		if ( this.elementName.isEmpty() ) {
 			throw new IllegalArgumentException( "Element name must not be empty." );
 		}
-		this.suffix = suffix.trim();
+		this.suffix = suffix;
 	}
 
 	public PathElement( final String elementName ) {
-		this( elementName, "" );
-	}
-
-	public PathElement( final String elementName, final int suffix ) {
-		this( elementName, "" + suffix );
+		this( elementName, 0 );
 	}
 
 	public String getElementName() {
 		return elementName;
 	}
 
-	public String getSuffix() {
+	public int getSuffix() {
 		return suffix;
 	}
 
 	@Override
 	public String toString() {
-		if ( "".equals( suffix ) ) {
-			return elementName;
-		} else {
-			return elementName + "[" + suffix + "]";
-		}
+		return elementName + "[" + suffix + "]";
 	}
 
 	@Override
 	public int hashCode() {
 		final int prime = 31;
 		int result = 1;
 		result = prime * result + (elementName == null ? 0 : elementName.hashCode());
-		result = prime * result + (suffix == null ? 0 : suffix.hashCode());
+		result = prime * result + (Integer.toString( suffix )).hashCode();
```

Comment:
```suggestion
		result = prime * result + int;
```

If in doubt you can also look at what the IDE generates.